docs/zh-Hans/Redis-Cache.md

```diff
@@ -1 +1,45 @@
-TODO...
\ No newline at end of file
+# Redis 缓存
+
+ABP Framework [缓存系统](Caching.md) 拓展了 [ASP.NET Core 分布式缓存](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed). 因此, 标准 ASP.NET Core 分布式缓存支持的 **任何提供程序** 都可以在您的应用程序中使用，并且可以像 **微软文档** 那样进行配置.
```

Comment:
你 instead of 您

Reply:
It will be better if replace `https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed` with `https://docs.microsoft.com/zh-cn/aspnet/core/performance/caching/distributed`